dash/dash-renderer/src/actions/callbacks.ts

```diff
@@ -226,8 +226,16 @@ function refErr(errors: any, paths: any) {
 const getVals = (input: any) =>
     Array.isArray(input) ? pluck('value', input) : input.value;
 
-const zipIfArray = (a: any, b: any) =>
-    Array.isArray(a) ? zip(a, b) : [[a, b]];
+const zipIfArray = (a: any, b: any) => {
+    if (Array.isArray(a)) {
+        // For client-side callbacks with multiple Outputs, only return a single dash_clientside.no_update
+        if (b?.description === 'Return to prevent updating an Output.') {
```

Comment:
Wouldn't `b === window.dash_clientside.no_update` be the same comparison since it's "singleton-like" variable?